news/_posts/2020-07-16-journey-to-improve-the-time-to-interactive-metric.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "A journey to improve the Time To Interactive metric in Classic Theme"
+subtitle: "When the lazy loading allow us to improve the LightHouse Score"
+date: 2020-07-16 9:00:00
+authors: [Progi1984]
+image: /assets/images/theme/meta-logo-build.png
+icon: icon-lab
+tags:
+- development
+- ux
+- theme
+- 1.7.8
+---
+
+The project [Core Web Vitals](https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html) from Google has announced that loading time and performance will have an impact on the website ranking in search results. In advices offered by Google, they talked about lazy loading. The lazy loading should be available on Classic Theme in version 1.7.8 thanks to [Progi1984](https://github.com/Progi1984) and it will definitely improve both the user experience and the loading time on your site!
```

Comment:
```suggestion
The project [Core Web Vitals](https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html) from Google has announced that loading time and performance will have an impact on the website ranking in search results. Google mentions lazy loading in their suggestions. Lazy loading will be available on Classic Theme in version 1.7.8 thanks to [Progi1984](https://github.com/Progi1984) and it will definitely improve both user experience and loading time on your site!
```